website/docs/markdown-features.mdx

```diff
@@ -420,9 +420,6 @@ The following example is how you can have multi-language code tabs in your docs.
 
 And you will get the following:
 
-import Tabs from '@theme/Tabs';
```

Comment:
No need to redeclare because markdown above alrd has `npm2yarn` meta which will import the Tabs component for us